std_msgs/Header header
ParticleMean[] particle_array
